Code/Source/Lidar/LidarRaycaster.cpp

```diff
@@ -84,12 +84,13 @@ namespace RGL
     {
         ValidateRayRange(range);
         m_range.second = range;
-        m_graph.ConfigureRayTraceNode(range);
+        m_graph.ConfigureRayRangesNode(m_range.first, m_range.second);
```

Comment:
If you configure min_range, this [if-statement](https://github.com/RobotecAI/o3de-rgl-gem/blob/rglv15_update/Code/Source/Lidar/LidarRaycaster.cpp#L179) never happens. RGL assigns to non-hit points a `NON_HIT_VALUE` which is the max value of the float (in both cases: when the object is too far and too close).

I suggest configuring RGL with `min_range` equal to zero to be able to distinguish whether the hit point is too close or too far. In the future, we will try to resolve this problem in the RGL.

*Remember to leave relevant comment in the code*

Reply:
Thanks! Please, create an issue for that on the RGL repository.

Reply:
Added to our internal board :white_check_mark: